poc/vidpf.py

```diff
@@ -154,8 +154,19 @@ def eval(cls,
                 (seed, ctrl, y, pi_proof) = prefix_tree_share.get(
                     (node, current_level))
 
+        # Compute the Aggregator's share of `beta`.
+        y0 = prefix_tree_share[(0, 0)][2]
+        y1 = prefix_tree_share[(1, 0)][2]
+        beta_share = vec_add(y0, y1)[1:]  # first element is the counter
+        if agg_id == 1:
+            beta_share = vec_neg(beta_share)
+
+        # Compute the Aggregator's share of the counter.
+        counter_share = y0[0] + y1[0] + cls.Field(agg_id)
```

Comment:
I think this counter_share isn't getting returned by the VIDPF or included in the out_share?

Reply:
the `out_share` returned at the end of this function would include a share of  $1 \|| \beta$ if one of the prefixes is on the $\alpha$ path. At this point we're just checking that the programmed value is indeed $1$.